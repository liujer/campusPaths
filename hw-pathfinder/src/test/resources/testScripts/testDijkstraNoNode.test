# Tests the case where the nodes do not exist in graph
CreateGraph A
AddNode A one
AddNode A two

AddEdge A one two 2

FindPath A one three