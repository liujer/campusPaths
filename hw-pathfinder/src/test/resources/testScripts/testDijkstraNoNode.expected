# Tests the case where the nodes do not exist in graph
created graph A
added node one to A
added node two to A

added edge 2.000 from one to two in A

unknown node three